### ISSUE TITLE
Configurable long press timeout

### ISSUE THREAD

**Issue by chrisboyle** [chrisboyle/sgtpuzzles#368]
Requested by Brian C, who wanted "something around 200ms". It's entirely possible people want a different value in Puzzles versus everywhere else, and they can't typically adjust the system default anyway.

**Comment by chrisboyle** [comment]
Daniel DiCato writes:
> adjustable length for what registers as a long press. I got the idea from SwiftKey - it has a "Long Press Duration" slider from 100ms to 1600ms. My personal preference is about 150ms, so that I can do repetitive operations much faster than with default settings.

**Comment by chrisboyle** [comment]
Luke Wolff writes:
> Please add a setting to allow users to change the long press duration. I keep accidentally letting go way before the required duration since I am used to a long press duration of 150ms for my keyboard app. Thanks!

**Comment by chrisboyle** [comment]
Trevos writes:
> Please allow the option to customize the length it takes to place a mine. It feels too long and occasionally makes me unintentionally open a mine that I wanted to flag.

**Comment by chrisboyle** [comment]
Jacqueline Liu writes:
> Long press time needs to be shorter, or at least user-configurable. As you can see, I died several times in situations in which it's very obvious that the grid has a mine, because I lifted my finger too soon.

**Comment by chrisboyle** [comment]
+1 from Filip Burlacu

**Comment by chrisboyle** [comment]
On my Pixel 6 Pro the delay is at least somewhat configurable, but seemingly only to make it longer than the default. :-(

This is in the system settings -> Accessibility.

![Screenshot_20220328-222616](https://user-images.githubusercontent.com/127305/160490249-2056735f-4ecd-400a-86cd-e7d69dc7364f.png)

**Comment by obar** [comment]
I went in and set my long press delay to "short" as suggested above, but it's still too long a timeout for playing Mines. I'm selecting mines that I'm trying to flag (and losing the game). I could see the [long press feedback feature](https://github.com/chrisboyle/sgtpuzzles/issues/364) being helpful here.

Like Daniel, I've also got SwiftKey with a customized long-press time (185ms).

I'm looking forward to seeing this change!
